formidling/src/main/resources/pdfgen/templates/ungdomsprogramytelse/endring_inntekt_uten_reduksjon.hbs

```diff
@@ -0,0 +1,17 @@
+{{#> felles/base}}
+    {{#> felles/seksjon/hoved }}
+        <h1>Vi har ikke endret ungdomsprogramytelsen din</h1>
+        {{#each perioder}}
+            <p>Du har gitt oss beskjed om at du hadde inntekt i perioden {{> felles/periode-inline periode}}.</p>
+            <p>Vi har kontrollert inntekten din for denne perioden og kommet frem til at du ikke får redusert ungdomsprogramytelsen din.</p>
```

Comment:
eg ville tenkt at "vurdert" er bedre enn "kontrollert" her. Kontroll høres strengere ut enn vurdering, og dette er jo gode nyheter for bruker.

Reply:
vi begynte med vurdert og endret til kontrollert 😅